grml-debootstrap

```diff
@@ -298,7 +298,12 @@ cleanup() {
 
   if [ -n "${ORIG_TARGET}" ] ; then
     einfo "Removing loopback mount of file ${ORIG_TARGET}."
-    kpartx -d "${ORIG_TARGET}" ; eend $?
+    kpartx -d "${ORIG_TARGET}"
+    if dmsetup ls | grep -q ${LOOP_DISK}; then
```

Comment:
shellcheck is right, the variable should be quoted :) See https://travis-ci.org/grml/grml-debootstrap/jobs/408132398

Reply:
Yeah, saw that and should be fixed in the following commit.